### ISSUE TITLE
[469b] Music not play if music name not match package name

### ISSUE THREAD

**Issue by SeriousBuggie** [OldUnreal/UnrealTournamentPatches#909]
Reproduce:
1. Start in UT [MissedMusic.zip](https://github.com/OldUnreal/UnrealTournamentPatches/files/9006197/MissedMusic.zip)

Expected result: You hear `Music'Credits.Trophy'`
Actual result: Silence.

If you rename file to Trophy.umx it will play.

Affected: OpenAL, Cluster, Galaxy.

**Comment by SeriousBuggie** [comment]
Hm. When you save level Song goes to None.
![img](https://user-images.githubusercontent.com/70026933/176329793-637e9ee3-3741-4095-9725-741d2842ae61.png)
![img](https://user-images.githubusercontent.com/70026933/176329845-3bb24ae5-0406-4f23-b7f4-be6221294236.png)

So now steps for reproduce goes to:
1. Load in UnrealEd [MissedMusic.zip](https://github.com/OldUnreal/UnrealTournamentPatches/files/9006197/MissedMusic.zip)
2. Open Music Credits.umx (stock file)
3. Fill LevelInfo.Song with `Music'Credits.Trophy'`
4. Save map.
5. Open map again.

Expected result: LevelInfo.Song = `Music'Credits.Trophy'`
Actual result: LevelInfo.Song = None

**Comment by SeriousBuggie** [comment]
Nope. LevelInfo.Song saved into file. But not able be properly loaded...
UTPT show: info here.
![img](https://user-images.githubusercontent.com/70026933/176331477-dd4c2b90-2eb2-4152-9eaf-3112dbde0721.png)

Also issue exists in v469c, v469b, v436.

**Comment by SeriousBuggie** [comment]
I even see in log load this package after load map:
`DevLoad: Loading: Package Credits`
(OFC for see this need disable `Suppress=DevLoad` in UT.ini)

**Comment by SeriousBuggie** [comment]
It is not about mismatch song name and package. It is About exactly Credits.Trophy music file. When it is named as Credits.
If rename it to Credits_test - it is work fine.

**Comment by SeriousBuggie** [comment]
It is happens because in stock UT exists credits.utx and credits.umx.
So on try load level it goes for load credits.utx, not found music here and set it to None.

**Comment by SeriousBuggie** [comment]
Possible need make some workaround on native part, especially for this files, when try load it. Since it stock files.

Or not touch it. Miss music not so big deal as miss textures.

**Comment by stijn-volckaert** [comment]
This is not easy to fix so don't expect this one for 469c
